src/main/java/org/folio/rest/validator/TitleParametersValidator.java

```diff
@@ -0,0 +1,42 @@
+package org.folio.rest.validator;
+
+import org.folio.rest.model.Sort;
+
+import javax.validation.ValidationException;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Objects;
+
+public class TitleParametersValidator {
+
+  private static final List<String> SELECTED_VALUES = Arrays.asList("true", "false", "ebsco");
+  private static final List<String> TYPE_VALUES = Arrays.asList("audiobook", "book", "bookseries", "database", "journal", "newsletter", "newspaper", "proceedings", "report", "streamingaudio", "streamingvideo", "thesisdissertation", "website", "unspecified");
+
+  /**
+   * @throws ValidationException if validation fails
+   */
+  public void validate(String filterSelected, String filterType, String filterName, String filterIsxn, String filterSubject,
```

Comment:
I am curious about something here: 
1. From UI, we are passing filter[selected] - correct? how is that getting converted to filterSelected?

Reply:
yes, we're passing filter[selected] I think it is mapped to filterSelected correctly because filterSelected  is annotated with  @QueryParam("filter[selected]")  in the generated interface